### PR TITLE
Added doc for LPS :)

### PR DESCRIPTION
**Pull request by BNMetrics**

### CODE REVIEW

File: docs/quickstart.html [hunk 1]
```diff
@@ -25,6 +25,17 @@ <h2>Shell completion</h2>
     <p>The script will complete subcommands and flags but most relevantly can also complete
       build labels by reading BUILD files for you.</p>
 
+    <h2>Buildfile Language Protocol Server(LPS)</h2>
+    <p>Please ships with a language server for build files. LPS follows standard
+        <a href="https://langserver.org/">language server protocol</a>
+    from VS Code.</p>
+
+    <p>To use LPS, simply download the language server protocol plugin for your favorite editor,
+    and run the binary at <code>~/.please/build_langserver</code></p>
```
Comment (peterebden) [inline docs/quickstart.html]: this should probably suggest `plz tool langserver` rather than the direct location?

File: docs/quickstart.html [hunk 2]
```diff
@@ -25,6 +25,17 @@ <h2>Shell completion</h2>
...
+
+    <p>Currently LPS supports auto-completion(this works with build labels too!), hover, goto defition,
```
Comment (peterebden) [inline docs/quickstart.html]: defition -> definition.

File: docs/quickstart.html (hunk 1 above)
Comment (BNMetrics) [reply docs/quickstart.html]: In that case, I will make it `plz tool lps` :)

File: src/tool/tool.go [hunk 3]
```diff
@@ -58,7 +58,7 @@ func matchingTools(config *core.Configuration, prefix string) map[string]string
 		"jarcat":      config.Java.JarCatTool,
 		"javacworker": config.Java.JavacWorker,
 		"junitrunner": config.Java.JUnitRunner,
-		"langserver":  path.Join(config.Please.Location, "build_langserver"),
```
Comment (peterebden) [inline src/tool/tool.go]: can you leave the old one in too please? at least for now, otherwise anyone using it now will break.

Comment (BNMetrics) [reply src/tool/tool.go]: sure! Done!

**Pull request merged**